services/iot-graphql-api/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  postgres:
```

Comment:
There's a custom Postgres dockerfile we're using. You can [copy this configuration](https://github.com/RedHat-Middleware-Workshops/rhtr-2020-api-mgmt-kafka-workshop/blob/master/services/docker-compose.yml#L32-L40).

Reply:
The reason for this is that it provides a pre-seeded DB for testing.

Reply:
I struggled to run it - credentials would not allow me to log in for some reason.